.github/workflows/build.yml

```diff
@@ -35,11 +35,11 @@ jobs:
     strategy:
       matrix:
         python-version: [3.7]
-        torch: [1.3.0, 1.5.0, 1.6.0, 1.7.0]
+        torch: [1.3.0, 1.5.1, 1.6.0, 1.7.0]
```

Comment:
Changes of this file could be dropped, since CI has been fixed.

Reply:
emmm, why there is no commit conflict?